torchprime/models/transact/transact_config.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
Could we add a run_model test that exercises a tiny version of the transact model? I looked around and only found a run_model test for a toy module but not sure if I missed something.

Reply:
done.